File: Assignment38/program4.c
    #include<stdio.h>
int DisplayI(int ino)
{
    int fact=1;
    while(ino!=1)
    {   
        fact = fact *ino;
        ino--;
    }
    return fact;
}
int DisplayR(int ino)
{
    static int fact=1;
    if(ino!=1)
    {   
        fact = fact *ino;
        ino--;
    }
    return fact;
}

int main()
{
    int iValue = 0,iRet=0;
    printf("Enter number: \n");
    scanf("%d",&iValue);
    iRet = DisplayR(iValue);
    printf("%d ",iRet);
}

File: Assignment31/program4.java
//
import java.lang.*;
import java.util.*;

class Bitwise
{
    public int ToggleBit(int iNo)
    {

        int iMask = 0X00000240; //7th bit 
        int iResult = 0;

        
        iResult = iNo ^iMask;
    
    return iResult;
    }
}

class program4

{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);  // 9th July

        System.out.println("Enter number ");
        int value = sobj.nextInt();


        Bitwise bobj = new Bitwise();
        int iret = bobj.ToggleBit(value);

        
        System.out.println("updated number "+iret);

    }
    
}




























